packages/components/src/DataList/components/DataListEmptyState/DataListEmptyState.css

```diff
@@ -8,3 +9,50 @@
   align-items: center;
   flex: 1;
 }
+
+.emptyStateWrapper h3 {
+  text-align: center;
```

Comment:
Couldn't figure out how to expose the `align` from `Typography` on `Heading` so here we are.